src/pages/en/guides/typescript.md

```diff
@@ -82,6 +83,18 @@ const { greeting = 'Hello', name } = Astro.props
 <h2>{greeting}, {name}!</h2>
 ```
 
+## Type checking
+
+To see type errors in your editor please make sure that you have the [Astro VS Code extention](/en/editor-setup/) installed. Please note that the `astro start` and `astro build` commands will transpile the code with esbuild, but will not run any type checking. To prevent your code from building if it contains TypeScript errors, change your "build" script in `package.json` to the following:
```

Comment:
```suggestion
To see type errors in your editor, please make sure that you have the [Astro VS Code extention](/en/editor-setup/) installed. Please note that the `astro start` and `astro build` commands will transpile the code with esbuild, but will not run any type checking. To prevent your code from building if it contains TypeScript errors, change your "build" script in `package.json` to the following:
```
Oops, spotted a missing comma! I'm just going to clean that up myself so we can merge this.  :)